### PR TITLE
Add some counts to studio UI

### PR DESCRIPTION
**Pull request by breville**
The studio UI now shows:
- lines of code counter on studio.code.org
- number of student accounts on student sign-up page
- number of teacher accounts on teacher sign-up page

### CODE REVIEW

**Comment by breville** [comment]
also @laurelfan you mentioned adding new indexes for the teacher/student account counts?

**Comment by laurelfan** [comment]
yep, one option could be to push those counts into the Properties table if we think that's a good place for things like this in general.

File: lib/cdo/properties.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      @table.insert(key:key, value:value.to_json)
+    else
+      @table.where(key:key).update(value:value.to_json)
+    end
+
+    value
+  end
+
+end
+PROPERTIES = Properties.new
```
Comment (laurelfan) [inline lib/cdo/properties.rb]: I'd recommend just making all of these methods class methods in Properties instead of initializing another constant.

**Comment by breville** [comment]
note to self: require 'cdo/properties' inside dashboard/config/application.rb.

**Comment by laurelfan** [comment]
_2072 – FOX Broadcasting television network will lose the rights to The Simpsons_

**Comment by breville** [comment]
@laurelfan here's an attempt at addressing the code review feedback.  thanks!

File: dashboard/config/locales/en.yml [hunk 2]
```diff
@@ -339,6 +341,7 @@ en:
     classroom_activity: 'Unplugged Activity'
   home:
     title: "Code Studio"
+    lines_of_code: "%{lines_count} LINES OF CODE WRITTEN BY STUDENTS"
```
Comment (laurelfan) [inline dashboard/config/locales/en.yml]: maybe do the capitalization in CSS?

File: lib/cdo/properties.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      @@table.insert(key:key, value:value.to_json)
+    else
+      @@table.where(key:key).update(value:value.to_json)
+    end
+
+    value
+  end
+
+  def self.get_user_metrics()
+    metrics = self.get(:about_stats)||{
```
Comment (laurelfan) [inline lib/cdo/properties.rb]: this local var isn't doing much since you just return it right away

File: lib/cdo/properties.rb [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      'number_students'=>5420082,
+      'number_teachers'=>124291
+    }
+    metrics
+  end
+
+end
+
+def fetch_metrics()
+  metrics = Properties.get(:metrics)||{
```
Comment (laurelfan) [inline lib/cdo/properties.rb]: same as above

**Pull request merged**